### ISSUE TITLE
Compatibility with vuex-i18n

### ISSUE THREAD

**Issue by lorro** [dobromir-hristov/vuelidate-error-extractor#21]
Hi,

To be honest, I remember you mentioning you hadn't prepared the plugin for vuex-i18n a while ago. Though, I wonder if you may have a suggestion for something I could try to get the result I need.

This is how I've setup vuex-18n right now.
When I set the default language to ``Vue.i18n.set('nl')` I get the correct NL translations. That works. But when I set the default language to EN (it should be), and we change to NL by a login of a Dutch user, the validation messages stay in the default language hence EN.

```js
// set the start locale to use, fetched from employee data
// default to english
Vue.i18n.set('en')
store.watch(() => store.getters['AUTH/user'], user => {
  if (user) {
    const locale = user.language || 'nl'
    Vue.i18n.set(locale)
    moment.locale(locale)
    return locale
  }
})
```
I actually had the impression it worked before, but perhaps I just looked wrong. I noticed this week I always get the english validation messages :) 

thanks!

**Comment by dobromir-hristov** [comment]
https://codesandbox.io/s/my6w8vv2ky
Works perfectly fine. I have a fix for the none translatable attributes, which will come out this weekend.

**Comment by lorro** [comment]
Thanks a lot for the example.
Did debug a lot, but it was apparently some silly stupid mistake.

I had VuelidateErrorExtractor attached to Vue like this
```js
Vue.use(Vuelidate)
Vue.use(VuelidateErrorExtractor, {
  messages: validationMessages,
  attributes: validationAttributes
})```

So without a i18n param. The idea behind that was I didn't have an i18n key in my translation files. Validation messages are on the same level as all my other messages.
Apparently I just had to add `i18n: some random string'` as a param and now it does work :)
Perhaps it wouldn't be a bad idea to allow `i18n: true` or `i18n: 'root'` or something like that?

Before I just wrapped Vue.i18n.translate functions around my messages, but actually it was not VuelidateErrorExtractor handling those translations.

Thanks again.
